spec/controllers/vacancies_controller_spec.rb

```diff
@@ -13,61 +13,39 @@
 
     context 'when parameters include syntax' do
       context 'search params' do
+        let(:expected_safe_values) { { keyword: 'Text' } }
         let(:params) do
           {
             keyword: "<body onload=alert('test1')>Text</body>",
             location: "<img src='http://url.to.file.which/not.exist' onerror=alert(document.cookie);>"
           }
         end
 
-        it 'passes only safe values to VacancyAlgoliaSearchBuilder' do
-          expected_safe_values = {
-            'keyword' => 'Text',
-            'location' => ''
-          }
-
-          expect(VacancyAlgoliaSearchBuilder).to receive(:new)
-            .with(expected_safe_values)
-            .and_call_original
-
+        it 'passes only safe values to Algolia::VacancySearchBuilder' do
+          expect(Algolia::VacancySearchBuilder).to receive(:new).with(expected_safe_values).and_call_original
           subject
         end
       end
 
       context 'sort params' do
+        let(:expected_safe_values) { { jobs_sort: 'Text' } }
         let(:params) do
-          {
-            jobs_sort: "<body onload=alert('test1')>Text</script>"
-          }
+          { jobs_sort: "<body onload=alert('test1')>Text</script>" }
         end
 
-        it 'passes sanitised params to VacancyAlgoliaSearchBuilder' do
-          expected_safe_values = {
-            'jobs_sort' => 'Text'
-          }
-
-          expect(VacancyAlgoliaSearchBuilder).to receive(:new)
-            .with(expected_safe_values)
-            .and_call_original
-
+        it 'passes sanitised params to Algolia::VacancySearchBuilder' do
+          expect(Algolia::VacancySearchBuilder).to receive(:new).with(expected_safe_values).and_call_original
           subject
         end
       end
 
       context 'search auditor' do
         let(:params) do
-          {
-            keyword: 'Teacher'
-          }
+          { keyword: 'Teacher' }
```

Comment:
I prefer one liner when is possible

```
let(:params) { { keyword: 'Teacher' } }
```

Reply:
Yep me too, will make sure I change all of them 👍